Expo SDK 52 Android builds are failing: Manifest merger failed

After upgrading to Expo SDK 52 and updating `react-native-fbsdk-next` to `13.4.0` my Android builds are failing with this error:
```
[RUN_GRADLEW] FAILURE: Build failed with an exception.
[RUN_GRADLEW] * What went wrong:
[RUN_GRADLEW] Execution failed for task ':app:processReleaseMainManifest'.
[RUN_GRADLEW] > Manifest merger failed : Attribute property#android.adservices.AD_SERVICES_CONFIG@resource value=(@xml/ga_ad_services_config) from [com.google.android.gms:play-services-measurement-api:22.0.1] AndroidManifest.xml:32:13-58
[RUN_GRADLEW]   	is also present at [com.facebook.android:facebook-core:18.0.0] AndroidManifest.xml:57:13-55 value=(@xml/ad_services_config).
[RUN_GRADLEW]   	Suggestion: add 'tools:replace="android:resource"' to <property> element at AndroidManifest.xml to override.
[RUN_GRADLEW] * Try:
[RUN_GRADLEW] > Run with --stacktrace option to get the stack trace.
[RUN_GRADLEW] > Run with --info or --debug option to get more log output.
[RUN_GRADLEW] > Run with --scan to get full insights.
[RUN_GRADLEW] > Get more help at https://help.gradle.org.
[RUN_GRADLEW] BUILD FAILED in 1m 18s
[RUN_GRADLEW] Error: Gradle build failed with unknown error. See logs for the "Run gradlew" phase for more information.
```

When I revert to `13.0.0` the Android build works fine.

Here is my `package.json`:

```json
{
  "name": "my-app",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    // ...
  },
  "dependencies": {
    "@amplitude/analytics-react-native": "^1.4.11",
    "@apollo/client": "^3.7.17",
    "@config-plugins/react-native-branch": "patch:@config-plugins/react-native-branch@npm%3A9.0.0#~/.yarn/patches/@config-plugins-react-native-branch-npm-9.0.0-f333c22663.patch",
    "@expo-google-fonts/libre-baskerville": "^0.2.3",
    "@expo-google-fonts/red-hat-display": "^0.2.3",
    "@expo-google-fonts/roboto-mono": "^0.2.3",
    "@expo/config-plugins": "~9.0.0",
    "@expo/react-native-action-sheet": "^4.0.1",
    "@formatjs/intl": "^2.10.0",
    "@gorhom/bottom-sheet": "^4.5.1",
    "@imgix/js-core": "^3.8.0",
    "@react-native-async-storage/async-storage": "1.23.1",
    "@react-native-community/netinfo": "11.4.1",
    "@react-native-community/slider": "4.5.5",
    "@react-native-firebase/analytics": "^20.1.0",
    "@react-native-firebase/app": "^20.1.0",
    "@react-navigation/bottom-tabs": "^6.5.8",
    "@react-navigation/material-top-tabs": "^6.6.5",
    "@react-navigation/native": "^6.1.7",
    "@react-navigation/native-stack": "^6.9.13",
    "@sentry/react-native": "~6.3.0",
    "@sentry/tracing": "^7.120.3",
    "@shopify/flash-list": "1.7.1",
    "@shopify/react-native-skia": "1.5.0",
    "@shopify/restyle": "^2.4.2",
    "@stream-io/flat-list-mvcp": "^0.10.3",
    "amplitude-js": "^8.21.9",
    "apollo3-cache-persist": "^0.14.1",
    "cheerio": "1.0.0-rc.12",
    "date-fns": "^2.30.0",
    "date-fns-tz": "^2.0.0",
    "emoji-datasource": "^15.0.1",
    "expo": "^52.0.27",
    "expo-apple-authentication": "~7.1.3",
    "expo-application": "~6.0.2",
    "expo-asset": "~11.0.2",
    "expo-auth-session": "~6.0.2",
    "expo-av": "~15.0.2",
    "expo-build-properties": "~0.13.2",
    "expo-clipboard": "~7.0.1",
    "expo-constants": "~17.0.4",
    "expo-contacts": "~14.0.3",
    "expo-crypto": "~14.0.2",
    "expo-dev-client": "~5.0.10",
    "expo-device": "~7.0.2",
    "expo-document-picker": "~13.0.2",
    "expo-file-system": "~18.0.7",
    "expo-font": "~13.0.3",
    "expo-haptics": "~14.0.1",
    "expo-image": "~2.0.4",
    "expo-image-manipulator": "~13.0.6",
    "expo-image-picker": "~16.0.4",
    "expo-linear-gradient": "~14.0.2",
    "expo-linking": "~7.0.4",
    "expo-localization": "~16.0.1",
    "expo-location": "~18.0.5",
    "expo-media-library": "~17.0.5",
    "expo-notifications": "~0.29.12",
    "expo-random": "~14.0.1",
    "expo-screen-orientation": "~8.0.4",
    "expo-secure-store": "~14.0.1",
    "expo-sharing": "~13.0.1",
    "expo-splash-screen": "~0.29.21",
    "expo-status-bar": "~2.0.1",
    "expo-store-review": "~8.0.1",
    "expo-system-ui": "~4.0.7",
    "expo-tracking-transparency": "~5.1.1",
    "expo-updates": "~0.26.13",
    "expo-web-browser": "~14.0.2",
    "firebase": "^10.1.0",
    "formik": "^2.4.2",
    "fuse.js": "^6.6.2",
    "google-libphonenumber": "^3.2.32",
    "graphql": "^16.8.1",
    "intl": "^1.2.5",
    "isomorphic-fetch": "^3.0.0",
    "lodash": "^4.17.21",
    "pusher-js": "^8.3.0",
    "react": "18.3.1",
    "react-native": "0.76.6",
    "react-native-appsflyer": "^6.15.3",
    "react-native-bidirectional-infinite-scroll": "^0.3.3",
    "react-native-branch": "^6.4.0",
    "react-native-cn-quill": "^0.7.18",
    "react-native-date-picker": "^4.2.13",
    "react-native-dialog": "^9.3.0",
    "react-native-drawer-layout": "^3.3.2",
    "react-native-fbsdk-next": "^13.4.0",
    "react-native-gesture-handler": "~2.20.2",
    "react-native-get-random-values": "~1.11.0",
    "react-native-hyperlink": "^0.0.22",
    "react-native-keyboard-aware-scroll-view": "^0.9.5",
    "react-native-localize": "^3.0.4",
    "react-native-mmkv": "2.12.2",
    "react-native-modal-datetime-picker": "^17.0.0",
    "react-native-network-logger": "^1.15.0",
    "react-native-open-graph-scraper": "^6.3.9",
    "react-native-pager-view": "6.5.1",
    "react-native-purchases": "^7.28.1",
    "react-native-reanimated": "~3.16.1",
    "react-native-render-html": "^6.3.4",
    "react-native-safe-area-context": "4.12.0",
    "react-native-screens": "~4.4.0",
    "react-native-shadow-2": "^7.0.8",
    "react-native-svg": "15.8.0",
    "react-native-svg-transformer": "^1.3.0",
    "react-native-tab-view": "^3.5.2",
    "react-native-web": "~0.19.13",
    "react-native-webview": "13.12.5",
    "reanimated-color-picker": "^2.3.2",
    "rrule": "^2.7.2",
    "stream-chat": "^8.26.0",
    "stream-chat-expo": "^5.27.0",
    "url-parse": "^1.5.10",
    "uuid": "^9.0.0",
    "yup": "^1.2.0"
  },
  "devDependencies": {
    "@babel/core": "^7.25.2",
    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
    "@babel/preset-typescript": "^7.24.1",
    "@bacons/xcode": "^1.0.0-alpha.6",
    "@config-plugins/apple-settings": "^0.0.5",
    "@graphql-codegen/add": "^5.0.0",
    "@graphql-codegen/cli": "^4.0.1",
    "@graphql-codegen/fragment-matcher": "^5.0.0",
    "@graphql-codegen/introspection": "^4.0.0",
    "@graphql-codegen/typed-document-node": "^5.0.1",
    "@graphql-codegen/typescript": "^4.0.1",
    "@graphql-codegen/typescript-apollo-client-helpers": "^2.2.6",
    "@graphql-codegen/typescript-document-nodes": "^4.0.1",
    "@graphql-codegen/typescript-graphql-files-modules": "^2.2.1",
    "@graphql-codegen/typescript-operations": "^4.0.1",
    "@graphql-codegen/typescript-react-apollo": "^3.3.7",
    "@testing-library/jest-native": "^5.4.3",
    "@testing-library/react-native": "^12.4.5",
    "@types/amplitude-js": "^8.16.5",
    "@types/fs-extra": "^11.0.3",
    "@types/glob": "^8.1.0",
    "@types/google-libphonenumber": "^7.4.23",
    "@types/inquirer": "^9.0.3",
    "@types/jest": "^29.5.12",
    "@types/linkify-it": "^3.0.4",
    "@types/lodash": "^4.14.195",
    "@types/react": "~18.3.12",
    "@types/url-parse": "^1",
    "@types/uuid": "^9.0.2",
    "@typescript-eslint/eslint-plugin": "^6.3.0",
    "@typescript-eslint/parser": "^6.3.0",
    "babel-jest": "^29.6.1",
    "babel-plugin-module-resolver": "^5.0.0",
    "cli-table3": "^0.6.3",
    "command-line-args": "^5.2.1",
    "dotenv": "^16.3.1",
    "eslint": "^8.46.0",
    "eslint-config-prettier": "^9.0.0",
    "eslint-config-universe": "^11.3.0",
    "eslint-plugin-import": "^2.29.1",
    "eslint-plugin-prettier": "^5.0.0",
    "inquirer": "^9.2.8",
    "jest": "^29.7.0",
    "jest-expo": "~52.0.3",
    "metro-react-native-babel-preset": "^0.77.0",
    "prettier": "^3.0.0",
    "prettier-eslint": "^15.0.1",
    "pretty-format": "^29.6.1",
    "react-test-renderer": "^18.2.0",
    "standard-version": "^9.5.0",
    "standard-version-expo": "^1.0.3",
    "ts-node": "^10.9.2",
    "typescript": "^5.6.2",
    "url": "^0.11.1"
  },
  "private": true,
  "packageManager": "yarn@4.5.0",
  "resolutions": {
    "semver": "7.6.3"
  }
}
```

Here is my `app.json` file:
```json
{
  "newArchEnabled": true,
  "android": {
    // ...
  },
  "assetBundlePatterns": [
    "**/*"
  ],
  "icon": "./src/assets/Icon_Production.png",
  "ios": {
    // ...
  },
  "jsEngine": "hermes",
  "name": "my-app",
  "owner": "me",
  "plugins": [
    "expo-apple-authentication",
    "expo-asset",
    "expo-font",
    "expo-localization",
    "expo-secure-store",
    "@react-native-firebase/app",
    "expo-tracking-transparency",
    "./plugins/android-clear-text-traffic-permitted",
    "./plugins/withAndroidVerifiedLinksWorkaround",
    "./plugins/withAppsFlyerFix.js",
    [
      "expo-build-properties",
      {
        "ios": {
          "useFrameworks": "static"
        }
      }
    ],
    [
      "expo-screen-orientation",
      {
        "initialOrientation": "DEFAULT"
      }
    ],
    [
      "react-native-appsflyer",
      {
        "shouldUseStrictMode": false
      }
    ],
    [
      "react-native-fbsdk-next",
      {
        "appID": "...",
        "displayName": "...",
        "clientToken": "...",
        "scheme": "...",
        "advertiserIDCollectionEnabled": true,
        "iosUserTrackingPermission": "Allowing this setting helps us personalize content.",
        "autoLogAppEventsEnabled": true,
        "isAutoInitEnabled": true
      }
    ],
    [
      "@config-plugins/react-native-branch",
      {
        "apiKey": "...",
        "iosAppDomain": "..."
      }
    ],
    [
      "@sentry/react-native/expo",
      {
        "organization": "...",
        "project": "..."
      }
    ]
  ],
  "runtimeVersion": {
    "policy": "appVersion"
  },
  "slug": "...",
  "userInterfaceStyle": "light",
  "version": "1.0.0",
  "web": {
    "favicon": "./src/assets/favicon.png"
  },
  "sdkVersion": "52.0.0",
  "platforms": [
    "ios",
    "android"
  ],
  "scheme": "my-app",
  "updates": {
    "url": "..."
  },
  "extra": {
    // ...
  }
}
```

--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -34,7 +34,7 @@
            
            https://github.com/thebergamo/react-native-fbsdk-next/issues/597
       -->
-      <manifest package="com.facebook.reactnative.androidsdk"
+<manifest package="com.facebook.reactnative.androidsdk"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:android="http://schemas.android.com/apk/res/android">
     <application>

--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -18,7 +18,23 @@
     CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -->
 
-<manifest package="com.facebook.reactnative.androidsdk"
+      <!--
+          This may generate a warning during your build:
+          
+           > property#android.adservices.AD_SERVICES_CONFIG@android:resource
+           > was tagged at AndroidManifest.xml:23 to replace other declarations
+           > but no other declaration present
+           
+           You may safely ignore this warning.
+           
+           We must include this in case you also use something else
+           that may also include this file, because if two modules include it, they
+           will collide and cause a build error if we don't set this one to take
+           priority via replacement.
+           
+           https://github.com/thebergamo/react-native-fbsdk-next/issues/597
+      -->
+      <manifest package="com.facebook.reactnative.androidsdk"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:android="http://schemas.android.com/apk/res/android">
     <application>

--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -18,6 +18,13 @@
     CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -->
 
-<manifest
-    package="com.facebook.reactnative.androidsdk">
+<manifest package="com.facebook.reactnative.androidsdk"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <application>
+        <property
+            android:name="android.adservices.AD_SERVICES_CONFIG"
+            android:resource="@xml/ad_services_config"
+            tools:replace="android:resource" />
+    </application>
 </manifest>

